service/popm/popm.go

```diff
@@ -435,24 +449,50 @@ func (m *Miner) BitcoinUTXOs(ctx context.Context, scriptHash string) (*bfgapi.Bi
 	return ir, nil
 }
 
+func (m *Miner) mineKnownKeystones(ctx context.Context) {
+	keystonesFailed := false
+
+	m.ForEachL2Keystone(func(ks hemi.L2Keystone) error {
```

Comment:
Why is this not a proper for loop?

Reply:
I wanted the `ForEachL2Keystone` to handle the locking and copying, I could pull it out and make it a for loop if you want

Reply:
changed it to a `for` loop 👍